integrations/phoneinfoga/compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+
+services:
+    phoneinfoga:
+      container_name: phoneinfoga
+      restart: unless-stopped
```

Comment:
please deploy only the REST API, as stated in [the doc](https://sundowndev.github.io/phoneinfoga/getting-started/usage/). The GUI is not required and it is heavy